app/Resources/translations/messages.nl_NL.xliff

```diff
@@ -435,13 +435,18 @@
         <source>ra.form.ra_verify_phone_number.text.challenge</source>
         <target>Code</target>
       </trans-unit>
+      <trans-unit id="a585bcd548874097afa817682427ed94ce28ab10" resname="ra.form.start_vetting_procedure.different_institution_error">
+        <jms:reference-file line="5">Resources/views/translations.html.twig</jms:reference-file>
+        <source>ra.form.start_vetting_procedure.different_institution_error</source>
+        <target>De gevonden verificatiecode hoort bij een gebruiker met een ander instituut dan die van de RA.</target>
```

Comment:
Was this translation provided by SURF? I think it can use some love; would be nice if it is directed at the user (the RA), rather than a factual statement. "De gevonden verificatiecode hoort bij een gebruiker behorende bij een ander instituut."